.eslintrc.base.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   plugins: ['@typescript-eslint', 'node'],
   extends: [
     'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
```

Comment:
LGTM with the caveat that we'll want to iterate on extending these rules to something we actually like so nyx can form the basis of a `yarn create` template